src/main/java/mekanism/common/tile/prefab/TileEntityStructuralMultiblock.java

```diff
@@ -48,6 +48,44 @@ public void setStructure(MultiblockManager<?> manager, Structure structure) {
         structures.put(manager, structure);
     }
 
+    @Override
+    public void removeStructure(Structure structure) {
```

Comment:
shouldnt this remove the map entry?

Reply:
I am not sure, my initial implementation before https://github.com/mekanism/Mekanism/pull/8051/commits/707069f3b1ffb988bfb54f88e36299a1f792ca09 had it removing it, but then when I thought about it more, realistically there are some cases where a multiblock forms and unforms, where I am not sure the structure becomes invalid. Like when revalidation is occurring (I think), so I wanted it to not have to remove the structure if it was still valid, and then recalculate it, compared to just be able to re-use it and only remove if necessary. Maybe this method would be better named `multiblockUnformed`?